openhands/runtime/plugins/agent_skills/file_ops/file_ops.py

```diff
@@ -239,6 +245,9 @@ def goto_line(line_number: int) -> None:
 
     CURRENT_LINE = _clamp(line_number, 1, total_lines)
 
+    if CURRENT_FILE is None:
+        _output_error('No file open. Use the open_file function first.')
+        return
```

Comment:
Is this intentional? I think maybe after _check_current_file() - we can assert CURRENT_FILE is not None?